packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -204,6 +204,30 @@ export interface BaseCustomStyles {
     root?: IStyle;
 }
 
+// @beta
+export interface BlockedMessage extends MessageCommon {
+    // (undocumented)
+    attached?: MessageAttachedStatus;
+    // (undocumented)
+    content?: string | false;
```

Comment:
Do we need false type here? Undefined should be good enough I guess

Reply:
By default if nothing is provided it will be 'undefined' and will follow the default behavior with default localization text. For it to not display the warningText/content, they will need to set it to false (eg. if they only want the linkText only but no warningText)